web/src/drafts.ts

```diff
@@ -654,13 +655,18 @@ export function format_draft(draft: LocalStorageDraftWithId): FormattedDraft | u
     }
 
     const emails = util.extract_pm_recipients(draft.private_message_recipient);
+    let no_recipients = false;
+    if (!draft.private_message_recipient) {
+        no_recipients = true;
+    }
```

Comment:
Since we don't need to get the emails or recipients for the empty string case, then we can do an early return here...

```javascript
    if (draft.private_message_recipient  === "") {
        // No users were set as DM recipients when the draft was created.
        return {
            draft_id: draft.id,
            is_stream: false,
            has_recipient_data: false,
            recipients: "",
            raw_content: draft.content,
            time_stamp,
            ...markdown_data,
        };
    }
    const emails = util.extract_pm_recipients(draft.private_message_recipient);
    const recipients = people.emails_to_full_names_string(emails);
}
```